locales/base/translation.json

```diff
@@ -354,6 +354,7 @@
   "restrictedRegion": "Limited support in region",
   "unsupportedPaymentMethod": "Paying with a {{paymentMethod}} is not supported in your region",
   "noProviders": "There are no providers available for {{digitalAsset}} in your region.",
+  "noPaymentMethods": "There are no payment methods available for {{digitalAsset}} in your region.",
```

Comment:
Wouldn't you also need to translate this line and add it to the various translation files?

Reply:
@anthonydaoud that will be done by our localization team!

Reply:
(separately from this PR)